core/src/main/java/tachyon/client/BlockOutStream.java

```diff
@@ -236,8 +232,8 @@ public void write(int b) throws IOException {
       throw new IOException("Out of capacity.");
     }
 
-    if (mBuffer.position() >= mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES,
-        Constants.MB)) {
+    if (mBuffer.position()
+        >= mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES, Constants.MB)) {
```

Comment:
When I was using
"
if (mBuffer.position() >=     mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES,		 
        Constants.MB)) {
"
and run "mvn package -DskipTests",
I got this "/apc999/core/src/main/java/tachyon/client/BlockOutStream.java:235:28: '>=' should be on a new line."

So I moved ">=" down to make style-checker happy and let me proceed.